resources/build/ci/help-keyman-com.sh

```diff
@@ -3,11 +3,15 @@
 ## START STANDARD BUILD SCRIPT INCLUDE
 # adjust relative paths as necessary
 THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
-. "${THIS_SCRIPT%/*}/../../resources/build/builder-basic.inc.sh"
+. "${THIS_SCRIPT%/*}/../../../resources/build/builder-basic.inc.sh"
```

Comment:
I think we'll also need to update the "Working directory" in 19.0 CI config steps that "Upload hew Keyman [Platform] help to help.keyman.com

to `keyman/resources/build/ci`

Reply:
ah, I guess all the 19.0 configs are already moved to teamcity scripts, so disregard :)